src/compat.jl

```diff
@@ -57,6 +59,18 @@ else
         args = [:(print("|  "); @show $a) for a in args]
         esc(:(println("E- ", $m); $(args...); nothing))
     end
+    Base.fetch(t::Task) = wait(t)
+
+end
+
+@static if isdefined(Base, :_sockname)
```

Comment:
Can you merge this into the `if VERSION >= v07` above?

Reply:
done